### PR TITLE
URL of OKD is docs.okd.io now

### PR DESCRIPTION
**Pull request by liangxia**
@bparees PTAL

### CODE REVIEW

**Comment by liangxia** [comment]
@pkubatrh PTAL

**Comment by pkubatrh** [comment]
Shouldnt this be pointing to https://docs.okd.io/latest/using_images/db_images/mariadb.html instead?

**Comment by liangxia** [comment]
Updated to mariadb now.

File: README.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ This repository contains Dockerfiles for MariaDB images for OpenShift and genera
 Users can choose between RHEL, Fedora and CentOS based images.
 
 MariaDB container is very similar to the MySQL container available at
-[https://github.com/sclorg/mysql-container](https://github.com/sclorg/mysql-container).
+[https://github.com/sclorg/mariadb-container](https://github.com/sclorg/mariadb-container).
```
Comment (pkubatrh) [inline README.md]: I think this was replaced by mistake? The link should still point to mysql here.

Comment (liangxia) [reply README.md]: Revert the line back.

**Comment by pkubatrh** [comment]
lgtm now, thanks!

**Pull request merged**